src/bank.rs

```diff
@@ -195,6 +196,14 @@ impl Bank {
             let account = accounts.entry(deposit.to).or_insert_with(Account::default);
             Self::apply_payment(deposit, account);
         }
+        {
```

Comment:
Can you move this block into a new `bpf_loader` fn.  Looks like it'll just require a `bank` reference.  This should limit the bank.rs touching